django_project/django_project/urls.py

```diff
@@ -16,14 +16,18 @@
 from django.contrib import admin
 from django.urls import path, include
 from django.conf import settings
+from django.conf.urls import url
 from django.conf.urls.static import static
+from user.views import LoginApiView, RegistrationApiView
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api/', include('user.urls')),
     path('', include('web_app.urls')),
-    path('auth/', include('rest_auth.urls')),
-    path('auth/registration/', include('rest_auth.registration.urls'))
+    path('auth/login/', LoginApiView.as_view()),
```

Comment:
All endpoints should be started with api/
You can add this to all your endpoints (except Django admin). Or you can create a variable and put here your URLs. And them include it to urlpatterns